.github/workflows/nodejs.yml

```
@@ -0,0 +1,39 @@
+name: 'tests'
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: 'ubuntu-latest'
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+
```

```suggestion
    strategy:
      matrix:
        node-version: [10.x, 12.x, 14.x]
      fail-fast: false 
```

You might want to use the `fail-fast`: `false` toggle here as well.
This makes it so that all test continue to run, even if one fails.
This can help to debug which version of Node is causing problems.